### ISSUE TITLE
"Skip to Main" link

### ISSUE THREAD

**Issue by buddy-harlow** [Bixal/bixal-site-drupal#239]
## Location: Across the entire site

- "Skip to Main" link is not visually perceivable. 
- Activating the link with a keyboard, does not actually take me to the main portion of the page.
- In-page anchor target not found (see screenshot below)

## Definition of Done
- First keyboard tab on the page should reveal the “Skip to main” link surrounded by a focus indicator
- When a user selects the “Skip to main content” link, they should be taken to the main content of the page (past the top nav menu)
